email2phonenumber.py

```diff
@@ -687,60 +947,60 @@ def setProxyList():
 bruteforce_parser.add_argument("-v", dest="verbose", action="store_true", help="verbose output")
 
 args = parser.parse_args()
-vars(args)["action"] = sys.argv[1] # Add missing param
+# Add missing param
+vars(args)["action"] = sys.argv[1]
```

Comment:
IndexError if executing the tool without arguments. It should default to show the help menu in that case (actually my bug :S)

Reply:
Solved 👍 
Now, logs the error and shows help menu.